flybywiresim/a32nx
# fix: show unlit button labels

## Summary of Changes

- Fixed button labels not showing when unlit
- Fixed white dots illuminating when ANN LT TEST is active
- Fixed SYS ON shown in blue instead of white
- Corrected the color of the two dots to better fit with the references

![Screenshot (80)](https://user-images.githubusercontent.com/71195324/110254972-ca8a9f80-7f91-11eb-8853-5574489077a4.png)

## Screenshots (if necessary)



### BEFORE:
![celabefor](https://user-images.githubusercontent.com/71195324/110255057-35d47180-7f92-11eb-9b85-29cfb9e197f9.png)
![backoverbefore](https://user-images.githubusercontent.com/71195324/110255134-95328180-7f92-11eb-98b8-b4aee0fca4dc.png)
![konzolabef](https://user-images.githubusercontent.com/71195324/110255155-a8dde800-7f92-11eb-8c95-bba6ca189b8e.png)
![sys](https://user-images.githubusercontent.com/71195324/110255176-b4c9aa00-7f92-11eb-9c22-4d72fd0d37f1.PNG)


### AFTER:

![cela](https://user-images.githubusercontent.com/71195324/110255048-2ead6380-7f92-11eb-9406-86e0e55a5ace.png)
![backover](https://user-images.githubusercontent.com/71195324/110255123-8ba91980-7f92-11eb-83d9-a7adcebe393a.png)
![konzola](https://user-images.githubusercontent.com/71195324/110255165-aed3c900-7f92-11eb-8ce8-3818ceb7753b.png)
![sysafter](https://user-images.githubusercontent.com/71195324/110255188-babf8b00-7f92-11eb-8dee-9a76aee054d1.PNG)

## References

![image0](https://user-images.githubusercontent.com/71195324/110255235-fc503600-7f92-11eb-9130-ed81390cb7b7.jpg)
![0848160](https://user-images.githubusercontent.com/71195324/110255247-1558e700-7f93-11eb-9913-e13a0fb3955d.jpg)
![Untitled-1](https://user-images.githubusercontent.com/71195324/110255312-54873800-7f93-11eb-975d-a07ca7b6917f.png)
![gear_pnls](https://user-images.githubusercontent.com/71195324/110255323-5a7d1900-7f93-11eb-80bc-e7d394ed3a42.jpg)

## Additional context

Approaches the same problem as @tyler58546 did with #3173, but does it individually for every button so that only the correct labels are visible.

Discord username (if different from GitHub): Imenes#8739

## Testing instructions

- Check that all buttons have visible labels when unlit
- Ensure that all buttons still work normally when pressed and when ANN LT TEST is active

Additional info:
This only applies to the labels that are already there, if you will go through the references you will see that there are some discrepancies and or missing labels like RAM Air, Ditching, EMR Call, etc. 
The OFF labels are a bit skewed on some buttons, this is all the same as it is in dev. Ignore those and any other preexisting issues not connected to this PR. 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
